src/codeflare_sdk/cluster/config.py

```diff
@@ -36,8 +36,12 @@ class ClusterConfiguration:
     name: str
     namespace: str = None
     head_info: list = field(default_factory=list)
-    head_cpus: typing.Union[int, str] = 2
-    head_memory: typing.Union[int, str] = 8
+    head_cpu_requests: typing.Union[int, str] = 2
```

Comment:
Would it be possible to check how the official Kubernetes Python client types resource requests and limits?

Reply:
The official resource [request/limits](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ResourceRequirements.md) is `dict(str, str)`

The SDK accepts both int and str and converts to str before setting the resources later in `generate_yaml.py`

Reply:
What's the plan to support different kind of accelerators, like AMD GPUs or Intel Gaudi?

Reply:
Kevin has this PR up for supporting different accelerators #531 I think it is pretty much there we should try get it in before next release